config/manifests/bases/node-healthcheck-operator.clusterserviceversion.yaml

```diff
@@ -14,6 +14,51 @@ spec:
       displayName: Node Health Check
       kind: NodeHealthCheck
       name: nodehealthchecks.remediation.medik8s.io
+      resources:
+      - kind: Nodes
```

Comment:
why we need node in the csv?

Reply:
scorecard wanted a declaration of what other crds we watch